weeder.dhall

```diff
@@ -1,10 +1,10 @@
 { roots =
   [ "^Main.main$"
+  , "^NITTA.Tests.tests$"
   , "^APIGen.main$"
   , "^Spec.main$"
   , "^Paths_.*"
   , "^Main.+Sync$", "^Main.+OnBoard$", "^Main.+Synchronization$"
-  , "^NITTA.Model.Microarchitecture.example$"
   , "^NITTA.Model.ProcessorUnits.IO.I2C.i2cUnit$"
```

Comment:
Do you review all this exceptions? Base on the weeder changelog, some of them can be removed.